### PR TITLE
v9-gui: improve labels for bulk actions

### PR DESCRIPTION
**Pull request by nqb**
# Description
- Handle case where bulk actions are use for **one** node or users
- Remove "Clear" in place of "Close" for security events

# Impacts
Nodes and Users searches and bulk actions

# Delete branch after merge
YES

### CODE REVIEW

**Comment by nqb** [comment]
@satkunas: to be complete, we should use `{securityEventCount}` in NodeSearch.vue.

**Comment by cgx** [comment]
We should do it the [proper way](https://kazupon.github.io/vue-i18n/guide/pluralization.html).

**Comment by nqb** [comment]
Thankss @cgx, I will leave it open for 9.1

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 1]
```diff
@@ -818,7 +818,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Closed security events on {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Closed security events on {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 2]
```diff
@@ -850,7 +850,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Registered {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Registered {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 3]
```diff
@@ -882,7 +882,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Deregistered {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Deregistered {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 4]
```diff
@@ -914,7 +914,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Reevaluated access on {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Reevaluated access on {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 5]
```diff
@@ -946,7 +946,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Restarted switch port on {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Restarted switch port on {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 6]
```diff
@@ -978,7 +978,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Refreshed fingerbank on {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Refreshed fingerbank on {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 7]
```diff
@@ -1010,7 +1010,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Applied role on {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Applied role on {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 8]
```diff
@@ -1042,7 +1042,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Applied bypass role on {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Applied bypass role on {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 9]
```diff
@@ -1074,7 +1074,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Applied security event on {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Applied security event on {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue [hunk 10]
```diff
@@ -1111,7 +1111,7 @@ export default {
             this.setRowVariant(index, convert.statusToVariant({ status: item.status }))
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Applied bypass VLAN on {nodeCount} nodes.', { nodeCount: items.length }),
+            message: this.$i18n.t('Applied bypass VLAN on {nodeCount} node(s).', { nodeCount: items.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue [hunk 11]
```diff
@@ -625,7 +625,7 @@ export default {
             }
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Applied {securityEventCount} security events for {userCount} users.', { securityEventCount: securityEventCount, userCount: this.selectValues.length }),
+            message: this.$i18n.t('Applied {securityEventCount} security event(s) for {userCount} user(s).', { securityEventCount: securityEventCount, userCount: this.selectValues.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue [hunk 12]
```diff
@@ -651,7 +651,7 @@ export default {
             }
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Closed {securityEventCount} security events for {userCount} users.', { securityEventCount: securityEventCount, userCount: this.selectValues.length }),
+            message: this.$i18n.t('Closed {securityEventCount} security event(s) for {userCount} user(s).', { securityEventCount: securityEventCount, userCount: this.selectValues.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue [hunk 13]
```diff
@@ -677,7 +677,7 @@ export default {
             }
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Registered {nodeCount} nodes for {userCount} users.', { nodeCount: nodeCount, userCount: this.selectValues.length }),
+            message: this.$i18n.t('Registered {nodeCount} node(s) for {userCount} user(s).', { nodeCount: nodeCount, userCount: this.selectValues.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue [hunk 14]
```diff
@@ -703,7 +703,7 @@ export default {
             }
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Deregistered {nodeCount} nodes for {userCount} users.', { nodeCount: nodeCount, userCount: this.selectValues.length }),
+            message: this.$i18n.t('Deregistered {nodeCount} node(s) for {userCount} user(s).', { nodeCount: nodeCount, userCount: this.selectValues.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue [hunk 15]
```diff
@@ -729,7 +729,7 @@ export default {
             }
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Applied role on {nodeCount} nodes for {userCount} users.', { nodeCount: nodeCount, userCount: this.selectValues.length }),
+            message: this.$i18n.t('Applied role on {nodeCount} node(s) for {userCount} user(s).', { nodeCount: nodeCount, userCount: this.selectValues.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue [hunk 16]
```diff
@@ -755,7 +755,7 @@ export default {
             }
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Applied bypass role on {nodeCount} nodes for {userCount} users.', { nodeCount: nodeCount, userCount: this.selectValues.length }),
+            message: this.$i18n.t('Applied bypass role on {nodeCount} node(s) for {userCount} user(s).', { nodeCount: nodeCount, userCount: this.selectValues.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue [hunk 17]
```diff
@@ -781,7 +781,7 @@ export default {
             }
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Reevaluated access on {nodeCount} nodes for {userCount} users.', { nodeCount: nodeCount, userCount: this.selectValues.length }),
+            message: this.$i18n.t('Reevaluated access on {nodeCount} node(s) for {userCount} user(s).', { nodeCount: nodeCount, userCount: this.selectValues.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue [hunk 18]
```diff
@@ -807,7 +807,7 @@ export default {
             }
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Refreshed fingerbank on {nodeCount} nodes for {userCount} users.', { nodeCount: nodeCount, userCount: this.selectValues.length }),
+            message: this.$i18n.t('Refreshed fingerbank on {nodeCount} node(s) for {userCount} user(s).', { nodeCount: nodeCount, userCount: this.selectValues.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue]: This should use `$tc` or `this.$18n.tc()`

File: html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue [hunk 19]
```diff
@@ -828,7 +828,7 @@ export default {
             this.setRowVariant(index, 'success')
           })
           this.$store.dispatch('notification/info', {
-            message: this.$i18n.t('Deleted {userCount} users.', { userCount: this.selectValues.length }),
+            message: this.$i18n.t('Deleted {userCount} user(s).', { userCount: this.selectValues.length }),
```
Comment (satkunas) [inline html/pfappserver/root/static.alt/src/views/Users/_components/UsersSearch.vue]: This should use `$tc` or `this.$18n.tc()`

**Comment by nqb** [comment]
@satkunas,

Is this PR still relevant for you ? If yes, I can do the substitution.

**Pull request merged**